test/extended/util/db_image_helpers.go

```diff
@@ -157,13 +157,21 @@ func WaitForQueryOutput(oc *CLI, d Database, timeout time.Duration, admin bool,
 		}
 		return false, nil
 	})
+	if err == wait.ErrWaitTimeout {
+		return fmt.Errorf("Timed out waiting for query: %q", query)
```

Comment:
because its sometimes very hard to deduce what exactly happened, this makes things a lot easier